smf/ntp/method/svc-site-ntp

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/ksh
+
+# This file and its contents are supplied under the terms of the
```

Comment:
Why are we licensing this script as CDDL? Is it mostly copied from illumos? If it's Oxide specific and new it should likely carry the standard Oxide license of MPL v2.0